input/syslogparser/syslogparser.go

```diff
@@ -114,40 +114,41 @@ func (parser *syslogParser) Parse(input []byte, timestamp time.Time) *base.LogRe
 	}
 
 	// parse the pri field, e.g. "<163>"
-	if ok, val, next := nextFieldBySpace(remaining); ok {
-		if val[len(val)-2:] != ">1" {
-			parser.onMalformed(record, fmt.Sprintf("invalid syslog pri '%s'", val), input)
-			return nil
-		}
-		pri := val[1 : len(val)-2]
-		priVal, err := strconv.Atoi(pri)
-		if err != nil {
-			parser.onMalformed(record, fmt.Sprintf("invalid syslog pri value '%s'", pri), input)
-			return nil
-		}
+	ok, val, next := nextFieldBySpace(remaining)
+	if !ok {
+		parser.onMalformed(record, "unfinished syslog", input)
+		return nil
+	}
 
-		// extract facility from pri
-		{
-			facility := priVal >> 3
-			if facility < 0 || facility >= len(syslogprotocol.FacilityNames) {
-				parser.onMalformed(record, fmt.Sprintf("invalid syslog facility %d", facility), input)
-				return nil
-			}
-			facilityName := syslogprotocol.FacilityNames[facility]
-			parser.fieldFacilityLocator.Set(fields, facilityName)
-		}
```

Comment:
why remove this curly braces and the previous one, but keep the curly braces of blocks below?

Reply:
The linter was complaining about levels of nesting in the ifs, not about the blocks themselves, so I kinda missed the latter ones.
Why use them anyway? To visually separate stuff and increase readability? This isn't something people usually do (at least in golang), so it kinda has an opposite effect in my case.